From 28877485e0eee182577adc2438145c884b74542a Mon Sep 17 00:00:00 2001
From: qmy <845319792@qq.com>
Date: Mon, 22 Jun 2020 01:59:31 -0700
Subject: [PATCH] 2nd branch master

---
 readme.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/readme.txt b/readme.txt
index d09914b..9885407 100644
--- a/readme.txt
+++ b/readme.txt
@@ -1,3 +1,4 @@
 this is a beginner
 this is zhe 1st change
 2nd stash test
+change readme in test1 branch
-- 
2.17.1

